.storybook/manager.js

```diff
@@ -0,0 +1,10 @@
+// TODO: A semi-shoddy hack to enable syntax highlighting for tsx in storybook-readme.
+// Probably should open a PR to put tsx in storybook-readme instead?
+// import 'prismjs/components/prism-tsx';
```

Comment:
The comments are about this import which is now also commented out. Should we remove 3 lines?